src/docs/product/issues/issue-details/performance-issues/consecutive-http/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Consecutive HTTP"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/consecutive-http/
+description: "Learn more about Consecutive HTTP and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Consecutive HTTP issues are created when a set of sequential HTTP spans consistently exceeds a duration threshold.
+
+## Detection Criteria
+
+The detector for this performance issue looks for a set of sequential, non-overlapping HTTP spans. It intentionally ignores non-HTTP spans when finding this set, as the browser commonly performs other asynchronous operations such as loading resources, between HTTP spans. 
+
+Once this sequence is found, the following must hold true for each HTTP span:
+
+- The HTTP method must be a GET, POST, DELETE, PUT, PATCH
+- The HTTP URL must not begin with _next/static/ or _next/data/
+
+Once these spans are found, the following must also hold true:
+
+- The duration of each HTTP span must exceed a threshold.
+- The time between each HTTP span must exceed a threshold.
+- The number of sequential HTTP spans must exceed a threshold.
+
+If Sentry doesn't detect a Consecutive HTTP issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
+
+## Span Evidence
+
+You can identify a Consecutive HTTP problem by four main aspects in the "Span Evidence" section:
+
+- Transaction name
+- Sequential HTTP Spans
+
+![Consecutive HTTP span evidence](span-evidence.png)
+
+You can view the span evidence by going to the "Issues" page in Sentry, selecting your project, selecting the Consecutive HTTP Queries error you want to examine, then scrolling down to the "Span Evidence" section in the "Details" tab.
+
+## Fixing Consecutive HTTP Issues
+
+The causes of consecutive HTTP issues vary here are some suggestions to help you resolve them.
+- If each HTTP span is independent from one another, consider parallelizing your HTTP spans so they are no longer sequential.
+- If you have control over the backend, consider optimizing your endpoints for this specific use case. You can investigate corresponding backend transactions to help identify potential sources of improvement.
+- The issue may also lie in the frontend, as span time does not always align with actual request time. This is because JavaScript doesn't always process an HTTP request as soon as its response is ready because it's asynchronous and may be processing other tasks.
```

Comment:
```suggestion
## Fixing Consecutive HTTP Issues

The causes of Consecutive HTTP issues vary, but here are some suggestions to help you resolve them:

- If your HTTP spans are independent, consider parallelizing them so they're no longer sequential.
- If you have control over the backend, consider optimizing your endpoints for this specific use case. You can investigate corresponding backend transactions to help identify potential improvements.
- Since span time doesn't always align with actual request time, the issue may also be a frontend one. This is because JavaScript doesn't always process an HTTP request as soon as its response is ready because it's asynchronous and may be processing other tasks.
```